### DISCUSSION TITLE
Multiple filters on WeekStartingTime (shortens the period of some indicators)

### DISCUSSION THREAD

**Discussion by hanshenrikbalshoej** [nbrown02/FlowViz#141] [Q&A] [answered]
Hi

I do not understand why there is several filters on WeekStartTime on eg "Cycle Time Scatter Plot" (also on all the other Cycle Time indicators and Productivity and Responsiveness). 

In the picture below, one of the filters (08-05-2024) is due to my adjustment to range (26 weeks), but I do not understand where the other 14-08-2024 filter comes from (what controls it). 

![image](https://github.com/user-attachments/assets/1b533c39-19e9-4d16-9efe-32e02dec2cea)

Can anyone help explain this?

Best 

Hans Henrik

**Comment by nbrown02** [comment]
Hey Hans,

Thanks for pointing this out. I just did a check and I can see the same issue :( 

I've created an issue for this, as it is a bug. The dark page slicers are persisting across dark AND light pages. Tbh, I had plans to remove the dark page as it was creating an overhead I could do without...so this has given me the nudge to fix it!

**Comment by hanshenrikbalshoej** [comment]
It seems to work in version from November 21st

**Reply by nbrown02** [reply, marked as answer]
Yes it was fixed as a result of your feedback, so thank you :)
